maven-artifact/src/main/java/org/apache/maven/artifact/repository/Authentication.java

```diff
@@ -21,6 +21,7 @@
 
 /**
  * Authentication
+ * <br>May 2020, MNG-5583 per endpoint PKI authentication
```

Comment:
This comment can be moved to release notes or a changelog, it doesn't have to be part of the Javadoc.